docs/update-locking.md

```diff
@@ -16,26 +16,59 @@ assumption that updates are locked (see [issue #20](https://github.com/resin-io/
 
 In supervisors >= v4.0.0 and any OS that is not Resin OS 1.X, the old lock location is completely ignored.
 
-### Taking the lock
+### Creating the lockfile
 
-Using the above-mentioned library, the lock can be acquired like in this CoffeeScript example:
+There are many different tools and libraries that provide proper lockfile functionality. There are many different tools and libraries to provide proper lockfile functionality and a few common examples are shown below.
```

Comment:
`There are many different tools and libraries that provide proper lockfile functionality` is repeated

Reply:
Duh, sorry!